contributing/development/configuring_an_ide/visual_studio.rst

```diff
@@ -15,7 +15,9 @@ Visual Studio requires a solution file to work on a project. While Godot does no
 with the solution file, it can be generated using SCons.
 
 - Navigate to the Godot root folder and open a Command Prompt or PowerShell window.
-- Run ``scons platform=windows vsproj=yes`` to generate the solution.
+- Run ``scons platform=windows vsproj=yes dev_build=yes`` to generate the solution with debug symbols.
+        The vsproj parameter signals that you want Visual Studio solution generated.
+        The dev_build parameter makes sure the debug symbols are included, to place breakpoints and step through code.
```

Comment:
I'd reword this as you can always "place breakpoints", they just wouldn't work without the debug symbols. :upside_down_face:
```suggestion
        The ``vsproj`` parameter signals that you want Visual Studio solution generated.
        The ``dev_build`` parameter makes sure the debug symbols are included, allowing to e.g. step through code using breakpoints.
```

I'm not sure about formatting/tabs, someone from the docs team should take care of that.

Reply:
done